src/kbc_modules/offline_fs_kbc/common.rs

```diff
@@ -50,9 +51,23 @@ pub fn load_keys(keyfile_name: &str) -> Result<Keys> {
         .collect()
 }
 
+pub fn load_resources(resources_file_name: &str) -> Result<Resources> {
+    let resources_json = fs::read_to_string(resources_file_name)?;
+    // Redact parsing errors to avoid side-channels
+    let encoded_resources: HashMap<String, String> = serde_json::from_str(&resources_json)
+        .map_err(|_| anyhow!("Failed to parse resources JSON file"))?;
+    encoded_resources
+        .iter()
+        .map(|(k, v)| match decode(v) {
+            Ok(resource) => Ok((k.clone(), resource)),
+            Err(_) => Err(anyhow!("Failed to decode resource")),
+        })
+        .collect()
+}
+
 pub mod tests {
     pub use super::*;
-
+    use crate::kbc_modules::ResourceName::*;
     pub use base64::encode;
```

Comment:
Similarly I would just import `base64` here.

Reply:
try only import `base64` here, meet follow issue
```
error[E0425]: cannot find function `encode` in this scope
   --> src/kbc_modules/offline_fs_kbc/common.rs:102:17
    |
102 |                 encode(KEY),
    |                 ^^^^^^
    |
   ::: /Users/liudali/.cargo/registry/src/github.com-1ecc6299db9ec823/base64-0.13.0/src/decode.rs:88:1
    |
88  | pub fn decode<T: AsRef<[u8]>>(input: T) -> Result<Vec<u8>, DecodeError> {
    | ----------------------------------------------------------------------- similarly named function `decode` defined here
    |
help: a function with a similar name exists
    |
102 |                 decode(KEY),
    |                 ~~~~~~
help: consider importing one of these items
    |
68  |     use base64::encode;
    |
68  |     use crate::kbc_modules::offline_fs_kbc::common::tests::base64::encode;
    |

```
So keep the code no change
```
pub use base64::encode;
```